### PR TITLE
swerve drive docstrings

### PR DESCRIPTION
**Pull request by Ishan1522**

### CODE REVIEW

File: src/main/java/frc/robot/BuildConstants.java [hunk 1]
```diff
@@ -1,17 +1,19 @@
...
   public static final String VERSION = "unspecified";
-  public static final int GIT_REVISION = 22;
-  public static final String GIT_SHA = "ee746c8ba2b3fd21fde3a8e41cdd2f213deee1be";
-  public static final String GIT_DATE = "2024-10-27 14:01:12 EDT";
-  public static final String GIT_BRANCH = "k";
-  public static final String BUILD_DATE = "2024-10-27 14:10:44 EDT";
-  public static final long BUILD_UNIX_TIME = 1730052644497L;
-  public static final int DIRTY = 1;
+  public static final int GIT_REVISION = 25;
+  public static final String GIT_SHA = "3b5012c7524895e7c511a707a351c6a1f9225018";
```
Comment (JacksonElia) [inline src/main/java/frc/robot/BuildConstants.java]: what is this

Comment (Ishan1522) [reply src/main/java/frc/robot/BuildConstants.java]: The commit id I believe; it is automatically generated when built

File: src/main/java/frc/robot/commands/drive/DriveCommandBase.java [hunk 2]
```diff
@@ -55,6 +53,11 @@ public void calculatePoseFromLimelight(int limelightNumber) {
 
       double distanceFromClosestAprilTag = vision.getLimelightAprilTagDistance(limelightNumber);
 
+      // Depending on how many april tags we see, we change our confidence as more april tags
+      // results in a much more accurate pose estimate
+      // TODO: check if this is necessary anymore with MT2, also we might want to set the limelight
```
Comment (Ishan1522) [inline src/main/java/frc/robot/commands/drive/DriveCommandBase.java:59]: check if what is necessary?

File: src/main/java/frc/robot/commands/drive/DriveCommandBase.java [hunk 3]
```diff
@@ -55,6 +53,11 @@ public void calculatePoseFromLimelight(int limelightNumber) {
...
+      //  so it only uses 1 april tag, if they set up the field wrong (they can set april tags +-1 inch I believe)
+      //  using multiple *could* really mess things up.
```
Comment (Ishan1522) [inline src/main/java/frc/robot/commands/drive/DriveCommandBase.java:62]: nope. see image 
CAD973F1-AAA1-4D48-8738-D2BADC7D0FF4
from [here](https://docs.limelightvision.io/docs/docs-limelight/pipeline-apriltag/apriltag-robot-localization).

File: src/main/java/frc/robot/commands/drive/DriveCommandBase.java (hunk 2 above)
Comment (Ishan1522) [reply src/main/java/frc/robot/commands/drive/DriveCommandBase.java:59]: @JacksonElia

**Pull request merged**